src/sentry/conf/server.py

```diff
@@ -1095,6 +1095,7 @@ def create_partitioned_queues(name):
     "organizations:issue-percent-display": False,
     # Enable team insights page
     "organizations:team-insights": True,
+    "organizations:team-insights-age-unresolved": False,
```

Comment:
Should we make this `inights-v2` or something so that we can use it for all new features?

Reply:
hmmm yes